packages/cadl-python/src/emitter.ts

```diff
@@ -1355,14 +1324,17 @@ function getNamespaces(context: DpgContext): Set<string> {
     return namespaces;
 }
 
-function emitCodeModel(context: EmitContext<EmitterOptions>) {
+function emitCodeModel(context: EmitContext<PythonEmitterOptions>) {
     const dpgContext = createDpgContext(context);
     const clientNamespaceString = getClientNamespaceString(dpgContext)?.toLowerCase();
     // Get types
     const codeModel: Record<string, any> = {
         namespace: clientNamespaceString,
         subnamespaceToClients: {},
     };
+    for (const model of getAllModels(dpgContext)) {
```

Comment:
After we `getAllModels`, we still need to iterate all the operation params and response, also iterate all the properties. Using it seems not so efficiency. I prefer use `isAzureCoreModel` in `getType` directly if I do not miss any further reasons.